docker/Dockerfile

```diff
@@ -61,13 +62,15 @@ RUN set -x \
     && tar -xzf "$DISTRO_NAME.tar.gz" \
     && mv bookkeeper-server-${BK_VERSION}/ /opt/bookkeeper/ \
     && rm -rf "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz.sha512" \
-    && pip install zk-shell \
-    && JAVA_HOME=$(dirname $(dirname $(readlink -f $(which java)))) \
-    && echo networkaddress.cache.ttl=1 >> $JAVA_HOME/conf/security/java.security \
-    && echo networkaddress.cache.negative.ttl=1 >> $JAVA_HOME/conf/security/java.security
+    && pip install zk-shell
 
 WORKDIR /opt/bookkeeper
 
+ENV JAVA_HOME /usr/lib/jvm/java-17-openjdk-$TARGETARCH
```

Comment:
can we use ```ENV JAVA_HOME $(dirname $(dirname $(readlink -f $(which java)))) ``` instead of ```ENV JAVA_HOME 
 /usr/lib/jvm/java-17-openjdk-$TARGETARCH```

Reply:
I have tested on my laptop with my personal image. It won't work.

```
└─[$] <> cat Dockerfile
FROM shoothzj/base:jdk17


ENV JAVA1_HOME $(dirname $(dirname $(readlink -f $(which java))))
```
```
docker run -it --rm test bash
root@f28cdaac85a6:/# echo $JAVA1_HOME
$(dirname $(dirname $(readlink -f $(which java))))
root@f28cdaac85a6:/#
exit
```

Reply:
+1 for not hard-coding the path.
Can we merge these two changed command into the previous one? The new one will generate more layers in docker